vm/ubpf_jit_x86_64.h

```diff
@@ -283,6 +296,15 @@ emit_load_imm(struct jit_state *state, int dst, int64_t imm)
     }
 }
 
+/* Emit instructions to perform a blind load of the immediate value. */
+static inline void
+emit_alu64_blind_load_imm_to_rcx(struct jit_state *state, uint64_t imm)
+{
+    uint64_t secret = (uint64_t)random();
+    emit_load_imm(state, RCX,  imm ^ secret);
+    emit_alu64_imm32(state, 0x81, 6, RCX, secret);
```

Comment:
Bug: Truncating secret to 32 bits from 64bits. This leaves the resulting RCX in an invalid state.